### ISSUE TITLE
XCode dependency

### ISSUE THREAD

**Issue by bytemybits** [slash-hq/slash#1]
Trying to install and got the following error:
```
slash: A full installation of Xcode.app is required to compile this software.
Installing just the Command Line Tools is not sufficient.
Xcode can be installed from the App Store.
```

I understand that for now this might be a requisite, but seems a little overkill to have the whole system dependent on XCode. I'll give it another go is this comes to fruition.
Props for the work though. 👍

**Comment by kondi-qordoba** [comment]
You are right. We will move away from full xcode. We are aiming to support Linux soon so we have no choice 👍

**Comment by konradrodzik** [comment]
You are right. We will move away from full xcode. We are aiming to support Linux soon so we have no choice 👍
